clang/tools/clang-offload-wrapper/ClangOffloadWrapper.cpp

```diff
@@ -341,6 +377,39 @@ class BinaryWrapper {
   std::vector<std::string> TempFiles;
 
 private:
+  std::string SymPropBCFiles;
+  std::unique_ptr<util::SimpleTable> SymProp_Table{nullptr};
+  LLVMContext SymProps_C;
```

Comment:
Please avoid use of Underscores in Variables and function names. LLVM coding standards asks us to use camel case for variable and function names where we do not use underscores.

Thanks

Reply:
Updated function and variable names.